paparazzi/paparazzi/src/main/java/app/cash/paparazzi/SnapshotImageModifier.kt

```diff
@@ -0,0 +1,29 @@
+package app.cash.paparazzi
+
+import app.cash.paparazzi.internal.ImageUtils
+import java.awt.image.BufferedImage
+
+/**
+ * Interface that allows access to the image from the bridge session for modification
+ */
+interface SnapshotImageModifier {
+
+  /**
+   * @param image The image from the bridge session to modify
+   * @return modified image
+   */
+  fun modifyImage(image: BufferedImage): BufferedImage
```

Comment:
The frameImage I mention above needs deviceConfig, should this be considered context and passed as a parameter to `modifyImage`?